### ISSUE TITLE
Contributing guidelines CONTRIBUTING.md is empty

### ISSUE THREAD

**Issue by realabja** [heartexlabs/label-studio#1554]
CONTRIBUTING.md file on master is incomplete, self explanatory!

**Comment by makseq** [comment]
@realabja Thank you very much for pointing this out! 

@smoreface Let's fix it? :slightly_smiling_face: 
We can use it as example: https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md

**Comment by smoreface** [comment]
Yes I have a template to use as a guideline too to work on this! I'll work on this next week :) Thanks @realabja :)

**Comment by realabja** [comment]
Thank you both!
